modules/persistent-storage-flexvolume-drivers.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// storage/persistent-storage/persistent-storage-flexvolume.adoc
+
+[id="flexvolume-drivers_{context}"]
+= About FlexVolume drivers
+
+A FlexVolume driver is an executable file that resides in a well-defined directory on all machines in the cluster, including both masters and nodes. {product-title} calls the FlexVolume driver whenever it needs to mount or unmount a volume represented by a `PersistentVolume` with `flexVolume` as the source.
+
```

Comment:
Openshift does not support installing flex driver on master nodes.

Reply:
Okay, changing it to read, "...n a well-defined directory on all nodes in the cluster."